bucket/crow-translate.json

```
@@ -0,0 +1,30 @@
+{
+    "version": "2.2.1",
+    "description": "A simple and lightweight translator that allows to translate and speak text.",
+    "homepage": "https://crow-translate.github.io/",
+    "license": "GPL-3.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/crow-translate/crow-translate/releases/download/2.2.1/crow-translate-2.2.1-win64.exe#/dl.7z",
+            "hash": "f90434b9691a3d2adbbc7ac033319385fa8f1fd7bdd0bf47badad654321e527d"
+        }
+    },
+    "bin": "crow.exe",
+    "shortcuts": [
+        [
+            "crow.exe",
+            "Crow"
+        ]
+    ],
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall*\" -Recurse",
```

Why post_install??

No need to keep it for whole process and linked.

```suggestion
    "pre_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall*\" -Recurse",
```

---

Do you mean as early as possible?